build/commands/scripts/sync.js

```diff
@@ -28,43 +28,30 @@ program
 const installDepotTools = (options = config.defaultOptions) => {
   options.cwd = config.braveCoreDir
 
-  util.run('git', ['submodule', 'sync'], options)
-  util.run('git', ['submodule', 'update', '--init', '--recursive'], options)
+  if (!fs.existsSync(config.depotToolsDir)) {
+    Log.progress('Install Depot Tools...')
+    fs.mkdirSync(config.depotToolsDir)
+    util.run('git', ['-C', config.depotToolsDir, 'clone', 'https://chromium.googlesource.com/chromium/tools/depot_tools.git', '.'], options)
+  }
 
+  Log.progress('Fixup Depot Tools...')
   // fixup depot tools after update
   if (process.platform !== 'win32') {
     util.run('git', ['-C', config.depotToolsDir, 'clean', '-fxd'], options)
     util.run('git', ['-C', config.depotToolsDir, 'reset', '--hard', 'HEAD'], options)
-    return
+  } else {
+    // On Windows:
+    // When depot_tools are already installed they redirect git to their own
+    // version which resides in a bootstrap-*_bin directory. So when we try to
+    // do git clean -fxd we fail because the git executable is in use in that
+    // directory. Get around that by using regular git.
+    let git_exes = util.run('where', ['git'], {shell: true})
+    let git_exe = '"' + git_exes.stdout.toString().split(os.EOL)[0] + '"'
+    if (git_exe === '""') git_exe = 'git'
+    util.run(git_exe, ['-C', config.depotToolsDir, 'clean', '-fxd'], options)
+    util.run(git_exe, ['-C', config.depotToolsDir, 'reset', '--hard', 'HEAD'], options)
   }
-  // On Windows:
-  // When depot_tools are already installed they redirect git to their own
-  // version which resides in a bootstrap-*_bin directory. So when we try to
-  // do git clean -fxd we fail because the git executable is in use in that
-  // directory. Get around that by using regular git.
-  let git_exes = util.run('where', ['git'], {shell: true})
-  let git_exe = '"' + git_exes.stdout.toString().split(os.EOL)[0] + '"'
-  if (git_exe === '""') git_exe = 'git'
-  util.run(git_exe, ['-C', config.depotToolsDir, 'clean', '-fxd'], options)
-  util.run(git_exe, ['-C', config.depotToolsDir, 'reset', '--hard', 'HEAD'], options)
-
-  // Get around the error in updating depot_tools on windows due to pylint.bat
```

Comment:
I don't think this is relevant anymore since it was fixed in upstream a long time ago